### DISCUSSION TITLE
lock views to prevent users from deleting/changing it (improved granular permissions)

### DISCUSSION THREAD

**Discussion by mesqueeb** [community/community#36388] [Projects and Issues]
as an administration I do not want all developers to be able to delete views / change the filters / columns of views and then save that so everyone's view changes...

now the only thing I can do is: tell my developers to not change views of the project and if they do, to not click the save button!

this level of trust is unmaintainable once a company grows...

**Comment by razvanavram** [comment]
+1 this. 
We have 4 projects managed through Github Projects with 3 developers and 1 tester. 
Like @mesqueeb I wouldn't want anyone to change/delete the views.

So a lock would be nice.

A perfect feature, though, would be to have "locked views" where everyone is on the same page and "personalized views" so the tester can organize the tasks as he wants and the other developers filter by their projects.

**Comment by hadarziv-army** [comment]
+1

**Comment by lmahoney1** [comment]
Running into this issue frequently. We have users who look at our project board to view the progress of requests they've made. Very common for them to click 'save' after adding a filter to find their specific issue which causes issues for other users visiting the board. Would really like to prevent certain users (or groups) from being able to save filters. Still want them to be able to change the filter but don't want them to be able to save / override the existing filters

**Comment by lubomirw** [comment]
+1
